### PR TITLE
Feat/secondary 2 and 2 wide

### PR DESCRIPTION
**Pull request by pdimova**
Added styles for wide and huge breakpoints:

wide:
Screenshot 2019-04-01 at 10 55 49

huge:
Screenshot 2019-04-01 at 10 54 59

### CODE REVIEW

File: packages/edition-slices/src/tiles/tile-am/index.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+} from "../shared";
+import styles from "./styles";
+
+const TileAM = ({ onPress, tile }) => (
+  <TileLink onPress={onPress} tile={tile}>
+    <View style={styles.container}>
+      <TileImage
+        aspectRatio={16 / 9}
+        style={styles.imageContainer}
+        uri={getCrop(tile.leadAsset || tile.article.leadAsset, "crop169")}
```
Comment (ValGeorgiev) [inline packages/edition-slices/src/tiles/tile-am/index.js]: Please check other tiles for this

File: packages/edition-slices/src/tiles/tile-an/index.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+} from "../shared";
+import styles from "./styles";
+
+const TileAN = ({ onPress, tile }) => (
+  <TileLink onPress={onPress} tile={tile}>
+    <View style={styles.container}>
+      <TileImage
+        aspectRatio={1}
+        style={styles.imageContainer}
+        uri={getCrop(tile.leadAsset || tile.article.leadAsset, "crop11")}
```
Comment (ValGeorgiev) [inline packages/edition-slices/src/tiles/tile-an/index.js]: same here, as above

File: packages/edition-slices/src/tiles/tile-am/index.js (hunk 1 above)
Comment (pdimova) [reply packages/edition-slices/src/tiles/tile-am/index.js]: tile.article.listingAsset added to expression

File: packages/edition-slices/src/tiles/tile-an/index.js (hunk 2 above)
Comment (pdimova) [reply packages/edition-slices/src/tiles/tile-an/index.js]: here also tile.article.listingAsset is added

**Comment by nehasri89** [comment]
@pdimova have you tested all the slices which were using `P` and `G` tiles just to make sure we haven't regressed?

**Comment by pdimova** [comment]
All tiles with roundels were checked (tileAN, tileG, tileP and tileAH) and it happened that tileAH (used in Comment Lead and Cartoon for tablet) was not updated.

**Comment by tuncaulubilge** [comment]
Needs a snapshot update, then we can merge this in

**Pull request merged**